### ISSUE TITLE
Add Single transferable vote as a option

### ISSUE THREAD

**Issue by bark** [cthit/VoteIT#56]
https://en.wikipedia.org/wiki/Single_transferable_vote
To example flag voting.

**Comment by Edholm** [comment]
TL;DR please

**Comment by lindskogen** [comment]
Also, shouldn't this be an issue for sektionsmötet and not for this project directly?

**Comment by Edholm** [comment]
@lindskogen clarification please

**Comment by bark** [comment]
https://www.youtube.com/watch?v=l8XOZJkozfI

**Comment by bark** [comment]
@lindskogen need to be a feature in the tool before you can vote whit it.

**Comment by lindskogen** [comment]
I meant that we should take it up for discussion first

**Comment by bark** [comment]
yes, we ask StyrIT if this is a good feture.

**Comment by Dr-Horv** [comment]
@krabban91 and the rest of StyrIT should discuss this.

**Comment by bark** [comment]
update: Styrit will handle this on theirs next meeting

**Comment by lindskogen** [comment]
@krabban91 do you have an update on this?

**Comment by krabban91** [comment]
Pretty sure that we brought this up on a meeting. Having the actual support for this would allow us to not vote as we did with the Hoodie competition (proposal vs. proposal until we have proposal vs rejection). 
To put this into system will definitely be a LOT harder than just accepting voteIT as "Closed voting". 
The meeting will not be able to trust this before verification rounds, just as voteIT still needs verification rounds on future Student division meetings. 

Keep this as a low priority for now.
